src/multitenant_sdk.test.js

```diff
@@ -174,4 +185,147 @@ describe('new MultitenantSharetribeSdk', () => {
       );
     });
   });
+
+  describe('clientData', () => {
+    it('returns client data', () => {
+      const { sdk } = createSdk();
+
+      return report(
+        sdk
+          .clientData()
+          .then(response => {
+            expect(response.data).toEqual(
+              expect.objectContaining({
+                client_id: '08ec69f6-d37e-414d-83eb-324e94afddf0'
+              })
+            );
+          })
+      );
+    });
+
+    it('client data request returns unauthorized', () => {
+      const { sdk, sdkTokenStore } = createSdk({ clientSecret: 'invalid' });
+      expect(sdkTokenStore.getToken()).toBeUndefined();
+  
+      // Anonymous token is stored
+      return report(
+        sdk.clientData()
+          .catch(e => {
+            expect(e).toBeInstanceOf(Error);
+            expect(e).toEqual(
+              expect.objectContaining({
+                status: 401,
+                statusText: 'Unauthorized',
+                data: 'Unauthorized'
+              })
+            );
+          })
+        );
+    });
+  
+    it('client data request returns not found', () => {
+      const { sdk } = createSdk({ clientSecret: 'valid-secret-invalid-hostname' });
+  
+      return report(
+        sdk.clientData()
+          .catch(e => {
+            expect(e).toBeInstanceOf(Error);
+            expect(e).toEqual(
+              expect.objectContaining({
+                status: 404,
+                statusText: 'Not Found',
+                data: 'Not Found'
+              })
+            );
+          })
+      );
+    });
+  });
+
+  describe('clientAuthData', () => {
+    it('returns client data and token by making a POST /token request, if no token is found in store', () => {
```

Comment:
Let's add a test that ensures that `client_data` does not get stored in the token store, which I think happens right now.

Reply:
Right, that needs to be excluded, I guess